src/main.ts

```diff
@@ -0,0 +1,37 @@
+import { getAllLaps, getLap } from './api-calls.ts';
+import showListData from './list.ts';
+import showMapData from './map.ts';
+import showRaceDetails from './race-details.ts';
+import './style.scss';
+
+const filename = 'SN2780_210722_11H00_NADINE_IDUBE_RACEWAY_16_5554.json';
```

Comment:
I'm going to change the filename on the API to prove a point. NEVER TRUST THE API.